backend/src/map_model.rs

```diff
@@ -861,16 +984,16 @@ impl Direction {
     // TODO strum?
```

Comment:
nit: you could remove this comment. I was considering a crate to autogen the to/from string, but it's overkill, especially now that this enum embeds a Direction